src/main/java/com/cburch/logisim/analyze/data/CoverColor.java

```diff
@@ -63,9 +63,8 @@ public Color getColor(int index) {
   }
 
   public Color getNext() {
-    index++;
     if (index < 0 || index >= colors.size()) index = 0;
```

Comment:
Given the fact `index` is never decremented, the condition like `index < 0` makes no much sense. Can we have it removed (IIRC, two places in this file)?

Also CHANGELOG entry please.

Reply:
The first one was okay, as it was not addressing the local variable (I renamed it), the second one has been fixed and I added a changelog entry.